backends-velox/src/test/scala/io/glutenproject/execution/VeloxDataTypeValidationSuite.scala

```diff
@@ -47,7 +47,8 @@ class VeloxDataTypeValidationSuite extends WholeStageTransformerSuite {
   protected def createDataTypeTable(): Unit = {
     TPCHTables = Seq(
       "type1",
-      "type2"
+      "type2",
```

Comment:
do we need to add some nested type tests to improve coverage later? E.g., ```{a, {b, c}}```. This can exercise related APIs [```getStruct```](https://github.com/apache/spark/blob/master/sql/catalyst/src/main/java/org/apache/spark/sql/catalyst/expressions/UnsafeRow.java#L421) at deeper level of complex type